config.js

```diff
@@ -74,7 +74,11 @@ config.http = {
     /**
      * South Port where the Ultralight transport binding for HTTP will be listening for device requests.
      */
-    port: 7896
+    port: 7896,
+    /**
+     * HTTP Timeout for the http command endpoint (in miliseconds).
+     */
+    timeout: 1000
```

Comment:
In order to keep backward compatibility, I'd suggest to include this new parameter commented, as shown in https://github.com/telefonicaid/iotagent-ul/pull/131/files#diff-e22b50e740e102c8de5e858dbcd7753d

Reply:
Because otherwise updating existing instances of IOTA-UL running in high-delay environement (e.g. average HTTP request time is about 2 seconds) will break after merging this PR.

Thus, default should be keeping "no timeout at all" that I undersatnd corresponds to the situation of not having the configuration defined so `if (config.getConfig().http.timeout)` evaluates to false. Is my interpretation correct?

Reply:
Yes, thank you for your clarification .

Reply:
Fixed in 3b54b57